src/libstd/keyword_docs.rs

```diff
@@ -1589,37 +1589,27 @@ mod type_keyword {}
 ///
 /// # Unsafe abilities
 ///
-/// Unsafe Rust adds some capabilities to Safe Rust:
-///
-/// - Dereference [raw pointers]
-/// - Implement `unsafe` [`trait`]s
-/// - Call `unsafe` functions (including foreign functions, some compiler
-/// intrinsics and the raw allocator)
-/// - Mutate [`static`]s (including [`extern`]al ones)
-/// - Access fields of [`union`]s
-///
-/// This seems rather arbitrary, why do I need a keyword for this ? The
-/// following section holds the answer.
-///
-/// ## Why do we need a keyword for this ?
-///
 /// **No matter what, Safe Rust can't cause Undefined Behavior**. This is
 /// referred to as [soundness]: a well-typed program actually has the desired
 /// properties. In Safe Rust, the language and the compiler do this mostly by
 /// themselves, whereas in Unsafe Rust the programmer must help. The
 /// [Nomicon][nomicon-soundness] has a more detailed explanation on the subject.
 ///
-/// You may have noted all uses of `unsafe` are not equivalent: some are here to
-/// note the existence of a contract the programmer must check, others are to
-/// say "I have checked the contract, go ahead and do this".
+/// Here are the abilities Unsafe Rust has in addition to Safe Rust:
```

Comment:
```suggestion
/// To ensure soundness, Safe Rust is restricted enough that it can be automatically checked.
/// Sometimes, however, it is necessary to write code that is correct for reasons
/// which are too clever for the compiler to understand. In those cases,
/// you need to use Unsafe Rust.
/// Here are the abilities Unsafe Rust has in addition to Safe Rust:
```